templates/account/account.py

```diff
@@ -57,17 +77,14 @@ def install(self):
         Install account
         '''
         
-        try:
-            self.state.check('actions', 'install', 'ok')
+        if self.installed:
```

Comment:
I think it should be just state check here without catching an error `self.state.check('actions', 'install', 'ok')`